app/controllers/concerns/idv/document_capture_concern.rb

```diff
@@ -33,13 +33,25 @@ def successful_response
     end
 
     # copied from Flow::Failure module
-    def failure(message, extra = nil)
-      flash[:error] = message
-      form_response_params = { success: false, errors: { message: message } }
+    def failure(message = nil, extra = nil)
```

Comment:
it appear that failure is only called once passing message as nil. i understand this is following the pattern for defining the failure method throughout the application; however, could it be worth eliminating this failure definition and above call

FormResponse.new(
{
  success: false,
  errors: make_error_hash(message),
  extra: extra}
}.compact
)